example/web/lib/main.dart

```diff
@@ -0,0 +1,117 @@
+import 'package:flutter/material.dart';
```

Comment:
This file is just the default `main.dart` generated by `flutter create .`. While `flutter create .` **is** in the instructions, it might still be useful to have this committed.

Reply:
I've removed these!